public/contribute/index_zh.ejs

```diff
@@ -1,3 +1,19 @@
+<%
```

Comment:
Some googling revealed that you have to define the functions like this:
<%
endsWith = function(str, suffix) {
    return str.indexOf (suffix, str.length - suffix.length) !== -1;
}   

languageDetect = function(entry, article) {
    if (endsWith(entry, "_zh")) {
        article.language = "zh";
    } else {
        article.language = "en";
    }
}
%>